src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -462,6 +505,54 @@ int encrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
         return len;
     }
 
+    int decrypt(ByteBuffer src, ByteBuffer dst) {
```

Comment:
Similar to above. It seems a bit strange to see decrypt(...) method in between encrypt(...) methods.